gcsweb/Makefile

```diff
@@ -148,3 +134,10 @@ container-clean:
 
 bin-clean:
 	rm -rf .go bin
+
+all-push: all-push-images push-manifest
+
+push-manifest:
```

Comment:
I was (for a while) keeping this in sync with https://github.com/thockin/go-build-template which did a lot of what you have done here, but just epsilon differently.  Would be nice to re-converge so fixes can be synced over time.

Reply:
ah, I'd forgotten about that. can update to address.